exporter/awsemfexporter/go.mod

```diff
@@ -7,17 +7,17 @@ require (
 	github.com/census-instrumentation/opencensus-proto v0.4.1
 	github.com/golang/protobuf v1.5.2
 	github.com/google/uuid v1.3.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/awsutil v0.64.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/cwlogs v0.64.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/metrics v0.64.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/resourcetotelemetry v0.64.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/opencensus v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/awsutil v0.66.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/cwlogs v0.66.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/metrics v0.66.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/resourcetotelemetry v0.66.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/opencensus v0.66.0
 	github.com/stretchr/testify v1.8.1
-	go.opentelemetry.io/collector v0.65.0
+	go.opentelemetry.io/collector v0.66.0
 	go.opentelemetry.io/collector/component v0.65.0
 	go.opentelemetry.io/collector/consumer v0.65.0
-	go.opentelemetry.io/collector/pdata v0.65.0
-	go.opentelemetry.io/collector/semconv v0.65.0
+	go.opentelemetry.io/collector/pdata v0.66.0
```

Comment:
Some are not updated, I think the update-otel script does not know about these modules.

Reply:
let me fix this 👍🏻

Reply:
I will  :)

Reply:
Updated the modules in this PR.

updated the script in a separate PR: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16458